test/testdata/packager/import-autocorrect/import-autocorrect.autocorrects.exp

```diff
@@ -0,0 +1,43 @@
+# -- test/testdata/packager/import-autocorrect/__package.rb --
+# frozen_string_literal: true
+# typed: strict
+# enable-packager: true
+
+class Package < PackageSpec
+  import Dep
```

Comment:
without the fix, we should see two `import Dep` statements

Reply:
When I check out the first commit in your pull request, which adds this test but has not yet fixed it, this test does not fail. So I don't think that we can actually use this as a regression test, and we'd want to make a CLI test.

But the existing CLI test that was updated in this PR seems like it already tests that, so I think it's fine to delete this test if you want.